src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -114,33 +114,21 @@ internal static class LoggingExtensions
                 logLevel: LogLevel.Debug,
                 formatString: "Bound metrics address: {address}");
 
-        private static readonly Action<ILogger, Exception> _metricUrlsUpdated =
-            LoggerMessage.Define(
-                eventId: new EventId(18, "MetricUrlsUpdated"),
-                logLevel: LogLevel.Warning,
-                formatString: $"Metric bindings changed. To host custom metrics over http set {ConfigurationPath.Combine(ConfigurationKeys.Metrics, nameof(MetricsOptions.AllowInsecureChannelForCustomMetrics))} to {true}");
-
-        private static readonly Action<ILogger, string, string, Exception> _metricUrlUpdated =
-            LoggerMessage.Define<string, string>(
-                eventId: new EventId(19, "MetricUrlUpdated"),
-                logLevel: LogLevel.Warning,
-                formatString: "Updated {originalUrl} to {newUrl} due to custom metrics.");
-
         private static readonly Action<ILogger, string, Exception> _optionsValidationFalure =
             LoggerMessage.Define<string>(
-                eventId: new EventId(20, "OptionsValidationFailure"),
+                eventId: new EventId(18, "OptionsValidationFailure"),
```

Comment:
We get a hallpass here since we haven't reached a stable release yet, but you really shouldn't change EventIds. That's like the public API surface for logs.

Reply:
Yup :-) it's the only reason why I'm changing them now.